src/main/java/ch/njol/skript/effects/EffFishingLure.java

```diff
@@ -0,0 +1,56 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.FishHook;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerFishEvent;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Apply Fishing Lure")
+@Description("Returns whether the lure enchantment should be applied to reduce the wait time.")
+@Examples({
+	"on fishing line cast:",
+		"\tapply lure enchantment"
+})
+@Events("Fishing")
+@Since("INSERT VERSION")
+public class EffFishingLure extends Effect {
+
+	static {
+		Skript.registerEffect(EffFishingLure.class,
+			"apply [the] lure enchantment",
+			"remove [the] lure enchantment");
```

Comment:
```suggestion
			"apply [the] lure enchantment bonus",
			"remove [the] lure enchantment bonus");
```
want to be clear we're not removing the enchantment itself.